### PR TITLE
DAOS-7174 doc: update to new pool list and tiering

### PR DESCRIPTION
**Pull request by johannlombardi**
Update output of pool list to match the new format.
A few changes related to the tiering patch.
Also fix several indenting nits.

doc-only: true

Signed-off-by: Johann Lombardi <johann.lombardi@intel.com>

### CODE REVIEW

File: doc/user/workflow.md [hunk 1]
```diff
@@ -25,7 +28,7 @@ each job.
 Another alternative use case is to emulate a parallel file system by creating
 one big pool with a single POSIX container to be accessed by all users.
 
-## `daos` Utility
+## `daos(1)` Utility
```
Comment (cdurf1) [inline doc/user/workflow.md:31]: What does the (1) signify here? Is it supposed to be a reference?

**Review by cdurf1 — APPROVED** [review]
Except for the clarification on why the parenthetical 1 in daos(1) utility, files look good to me.

File: doc/admin/pool_operations.md [hunk 2]
```diff
@@ -40,7 +42,7 @@ $ dmg pool create --help
       -P, --properties= Pool properties to be set
       -a, --acl-file=  Access Control List file path for DAOS pool
       -z, --size=      Total size of DAOS pool (auto)
-      -t, --scm-ratio= Percentage of SCM:NVMe for pool storage (auto) (default: 6)
+      -t, --tier-ratio= Distribution of pool storage allocation over storage tiers (auto) (default: 6)
```
Comment (tanabarr) [inline doc/admin/pool_operations.md:45]: alignment of columns has changed for this line and is now off, not sure whether that matters

File: doc/admin/pool_operations.md [hunk 3]
```diff
@@ -69,34 +71,126 @@ with redundancy enabled by default (pool service replicas on ranks 1-3).
...
+----     ----   ---- --------- --------
+tank     47 GB  0%   0%        0/32
+```
+
+This returns a table of pool labels (or UUIDs if no label was specified)
+with the following information for each pool:
+- the total pool size
+- the percentage of used space (i.e. 100 * used space  / total space)
+- the imbalance percentage indicating whether data distribution across
+  the difference storage nodes is well balance. 0% means that there is
```
Comment (tanabarr) [inline doc/admin/pool_operations.md:96]: balance->balanced

Comment (tanabarr) [reply doc/admin/pool_operations.md:96]: difference->different

File: doc/admin/pool_operations.md [hunk 4]
```diff
@@ -69,34 +71,126 @@ with redundancy enabled by default (pool service replicas on ranks 1-3).
...
+  no imbalance and 100% means that out-of-space errors might be returned
+  by some storage nodes while space is still available on some others.
+- the number of disabled targets (0 here) and the number of targets that
+  the pool was originally configured with.
```
Comment (tanabarr) [inline doc/admin/pool_operations.md:100]: I would end with "(total)." just to be explicit and not confuse with active.

File: doc/admin/pool_operations.md [hunk 5]
```diff
@@ -69,34 +71,126 @@ with redundancy enabled by default (pool service replicas on ranks 1-3).
...
+```
+
+This returns a table of pool labels (or UUIDs if no label was specified)
+with the following information for each pool:
+- the total pool size
+- the percentage of used space (i.e. 100 * used space  / total space)
+- the imbalance percentage indicating whether data distribution across
+  the difference storage nodes is well balance. 0% means that there is
+  no imbalance and 100% means that out-of-space errors might be returned
+  by some storage nodes while space is still available on some others.
```
Comment (tanabarr) [inline doc/admin/pool_operations.md:98]: some others->others

File: doc/admin/pool_operations.md [hunk 6]
```diff
@@ -69,34 +71,126 @@ with redundancy enabled by default (pool service replicas on ranks 1-3).
...
+- the number of disabled targets (0 here) and the number of targets that
+  the pool was originally configured with.
+
+The --verbose option provides more detailed information including the
+number of service replicate, the actual UUIDs and space distribution
```
Comment (tanabarr) [inline doc/admin/pool_operations.md:103]: replicate->replicas
actual->full

File: doc/admin/pool_operations.md [hunk 7]
```diff
@@ -69,34 +71,126 @@ with redundancy enabled by default (pool service replicas on ranks 1-3).
...
+        Free: 0, min:0, max:0, mean:0
+    Rebuild done, 10 objs, 1026 recs
+```
+
+The total and free sizes are the sum across all the targets whereas
+min/max/mean gives information about individual targets. A min value
+close to 0 means that one target is running out of space.
+
+NB: the Versioning Object Store (VOS) may reserve a portion of the
+SCM and NVMe allocations to mitigate fragmentation and for background
```
Comment (tanabarr) [inline doc/admin/pool_operations.md:158]: mitigate->mitigate against

File: doc/admin/pool_operations.md [hunk 8]
```diff
@@ -69,34 +71,126 @@ with redundancy enabled by default (pool service replicas on ranks 1-3).
...
+operations (e.g., aggregation, garbage collection). The amount of storage
+set aside depends on the size of the target, and may take up 2+ GB.
```
Comment (tanabarr) [inline doc/admin/pool_operations.md:160]: no comma needed

File: doc/admin/pool_operations.md [hunk 9]
```diff
@@ -69,34 +71,126 @@ with redundancy enabled by default (pool service replicas on ranks 1-3).
...
+Therefore, out of space conditions may occur even while pool query may not
```
Comment (tanabarr) [inline doc/admin/pool_operations.md:161]: Therefore, out->Out

File: doc/admin/pool_operations.md [hunk 10]
```diff
@@ -69,34 +71,126 @@ with redundancy enabled by default (pool service replicas on ranks 1-3).
...
+reveal min approaching zero.
```
Comment (tanabarr) [inline doc/admin/pool_operations.md:162]: reveal->show

**Review by tanabarr — APPROVED** [review]

File: doc/admin/pool_operations.md [hunk 11]
```diff
@@ -69,34 +71,126 @@ with redundancy enabled by default (pool service replicas on ranks 1-3).
...
+    - SCM:
+        Total size: 28GB
+        Free: 28GB, min:470MB, max:512MB, mean:509MB
+    - NVMe:
+        Total size: 56GB
+        Free: 28GB, min:470MB, max:512MB, mean:509MB
+    Rebuild busy, 75 objs, 9722 recs
+```
+
+Additional status and telemetry data are planned to be exported through
```
Comment (tanabarr) [inline doc/admin/pool_operations.md:180]: are->is

File: doc/user/container.md [hunk 12]
```diff
@@ -1,12 +1,14 @@
 # Container Management
 
-DAOS containers are datasets managed by the users. A container is the unit of
-snapshot and has a type. It can be a POSIX namespace, an HDF5 file or any other
-new application-specific data model. The chapter explains how to manager
-container, while the subsequent ones detail how to access a DAOS container from
-applications.
+DAOS containers (not to be confused with Linux containers) are datasets managed
+by the users. A container is the unit of snapshot and has a type. It can be a
+POSIX namespace, an HDF5 file or any other new application-specific data model.
```
Comment (tanabarr) [inline doc/user/container.md:5]: new is unnecessary here

File: doc/user/container.md [hunk 13]
```diff
@@ -1,12 +1,14 @@
...
+The chapter explains how to manager container, while the subsequent ones detail
```
Comment (tanabarr) [inline doc/user/container.md:6]: manager->manage a

**Review by tanabarr — APPROVED** [review]

File: doc/user/workflow.md (hunk 1 above)
Comment (johannlombardi) [reply doc/user/workflow.md:31]: It is referring to the section of the man page to check.

**Pull request merged**